lighthouse-core/report/v2/report-renderer.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/* eslint-env browser */
+
+class ReportRenderer {
+  /**
+   * @param {!Object} report
+   */
+  constructor(report) {
+    this._report = report;
+  }
+
+  /**
+   * @param {!Element} element
+   */
+  render(element) {
+    const pre = document.createElement('pre');
+    pre.innerText = JSON.stringify(this._report, null, 2);
+    element.appendChild(pre);
+  }
+}
+
+const renderer = new ReportRenderer(window.__LIGHTHOUSE_JSON__);
```

Comment:
Not that we need it right now, but can we keep usage out of this file and initiate in the .html? Thinking about the the Viewer extension use case:

https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/report/templates/report-template.html#L336